src/components/TradeWidget/TokenRow.tsx

```diff
@@ -139,8 +139,8 @@ interface Props {
 
 const BalanceTooltip = (
   <HelpTooltipContainer>
-    This balance reflects the amount deposited in the Exchange Wallet on Mesa. Only orders with a balance will be
-    considered for matching.
+    This balance reflects the amount deposited in the Exchange Wallet on Gnosis Protocol. Only orders with a balance
```

Comment:
```suggestion
    This balance reflects the amount deposited in the Exchange Wallet on Gnosis Protocol Web. Only orders with a balance
```

Reply:
@W3stside I did this one on purpose. The balance is in the contract, so I said Gnosis Protocol. I didn't want to refer to the UI. If I wanted to do that, also I wouldn't have "hardcode" it --> I could use the app name config